src/window/Window.cpp

```diff
@@ -237,18 +242,18 @@ void Window::InitWindowManager() {
             mWindowManagerApi = &gfx_dxgi_api;
             break;
 #endif
-#if defined(ENABLE_OPENGL) || defined(__APPLE__)
+#ifdef ENABLE_OPENGL
```

Comment:
i'm curious as to why this was an or in the first place, is `ENABLE_OPENGL` not set on mac?

Reply:
On mac ENABLE_OPENGL could be optional, but in that ifdef it was forced on